front-end/generate-task-lists.js

```diff
@@ -1,127 +1,160 @@
-const addTaskForm = document.querySelector('.task-form')
-const taskList = document.querySelector('.task-list')
+const addTaskForm = document.querySelector('.task-form');
+const taskList = document.querySelector('.task-list');
 
-const tasks = []
+let tasks = [];
+
+axios
+    .get('/tasks')
+    .then(response => {
+        // either empty array or array of tasks
+        tasks = Array.from(response.data);
```

Comment:
As the above comment says, the `response.data` variable is already an array, so there's no need to use `Array.from`